test/fit_tests/common/fit_common.py

```diff
@@ -363,7 +363,7 @@ def restful(url_command, rest_action='get', rest_payload=[], rest_timeout=None,
                                        verify=sslverify,
                                        )
         if rest_action == "binary-put":
```

Comment:
That is a good future enhancement. Let's write up a set of test enhancements in a story queue. We are going to be migrating off Jira. Should these be on Pivotal?